3rdPartyCypress/cypress/integration/worldServiceCoUkRedirect.js

```diff
@@ -1,37 +1,41 @@
 import services from '../../../src/server/utilities/serviceConfigs';
 
-Object.keys(services).forEach((service) => {
+Object.keys(services).forEach(service => {
   const notWSServices = [
     'news',
     'cymrufyw',
     'naidheachdan',
     'default',
     'scotland',
+    'archive', // not WS
+    'serbian', // variant with redirect - test needs to be updated
```

Comment:
Guessing this is needed to run the `cypress:3rdParty` cmd as part of CI & pass?

It's a little confusing to see ws variant services within a 'notWSServices' array - maybe a second array should be created to house the variants, and these two arrays then merged for the function:

```
 if (notWSServices.includes(service)) {
    return;	  
  }
```

Would be good to see the `tech-debt` ticket made at the same time to address this as a follow-up?

Reply:
> Guessing this is needed to run the cypress:3rdParty cmd as part of CI & pass?
Correct! I didn't want to spend too much time on it, just get it passing.

> Would be good to see the tech-debt ticket made at the same time to address the redirect tests failing for variants as a follow-up?
Good shout, it would be good to get it fixed up.